setup.py

```diff
@@ -22,7 +22,7 @@
 postgres_requires = sqlite_requires + ['psycopg2']
 mysql_requires = sqlite_requires + ['pymysql']
 
-omniscidb_requires = ['pymapd>=0.12.0']
+omniscidb_requires = ['pymapd>=0.12.0', 'pyarrow']
```

Comment:
you would need to add an entry in ci/deps/omnsidb.yml

Reply:
make sense. I am going to add that! thanks